docs/data/datanalytics/getting-started.md

```diff
@@ -0,0 +1,127 @@
+# Getting started
+
+Congratulations! You have either landed here because you want to learn data analysis or you want to learn about CSC's new data analysis infra - or both.
+
+Either case, you will have an interesting and giving journey ahead of you.
+
+To use the CSC environment you have to have a [CSC account](/accounts/creating-new-user-account/) and a [membership of a project](/accounts/joining-project/) that has Puhti [service enable](/accounts/adding-service-access-for-project/).
+
+After that you can [login](/connecting/) to Puhti environment.
+
+The Puhti environment supports a [wide list of applications](/apps/alpha/) for you to choose by default. Just [load](/computing/module-system/) the ones you need for your work or [build your own](/computing/module-system/#using-your-own-module-files).
+
+How data flows through different services during data analysis phases?
+
+[R for Data Science](https://r4ds.had.co.nz/introduction.html) by H. Wickham and G Grolemund describes the overall process of data analytics quite well. Below is an adaptation of that process picture adapted with different CSC services and how they are positioned in different phases.
+
+![Typical data science project with CSC environment](/img/datanalytics-data-in-process.png)
+
+<!-- @tolonenj: Describing how preprocessing data is often a lot of work (“data wrangling”)-->
+
+Let's get started!
+
+[TOC]
+
+## Starting with data that is stored in Allas
+
+Just a quick reminder what Allas is.
+
+> **Allas** is part of the CSC storage portfolio and accessible from anywhere on the Internet. It is a storage service to host data for a project lifetime. The data from Allas can also be shared via Internet.
+> -- <cite>[Allas](/data/Allas/introduction/)</cite>
+
+Few things worth to notice about Allas is that it is meant to store and share data over the timeline of a project, maximum of three to five years. Default quota for a project is 1 TB but please keep in mind that used storage consumes project's billing units. Also bear in mind that no backups are taken of the data located in Allas, so please take care of backups of your data.
```

Comment:
This also uses 1 TB for quota. Probably we should refer to the quota section for actual numbers so we don't get out-of-sync information